### ISSUE TITLE
Create and Revise Aligning Facility Security Level (FSL) and Authentication Section

### ISSUE THREAD

**Issue by clstmbrly** [GSA/ficam-pacs#8]
#### Description of Issue: ####

- Create a new major section:  “Aligning Facility Security Level (FSL) and Authentication”
- Under this section, combine 2 existing, major sections:  “Determine Facility Risk” (previously "Selecting Determining Facility Risk") and “Select Authentication Mechanism” (previously "Selecting the Appropriate Authentication Mechanism")
- Update this section with current information, practices, and graphics
- Restructure to improve the logical flow of information and revise content for clarity

#### Detailed Suggestions: ####

- Combine 2 existing, major sections:  “Determine Facility Risk” and “Select Appropriate Authentication Mechanism."  Include the subsection:  "Determine Authentication Factors" (previously "What Authentication Mechanisms Are Available?")
- Revise with the target audience in mind
- Add a purpose statement upfront
- Structure titles and content for logical flow of information
- Remove outdated practices and non-essential information. Update content with current information and practices.
- For the audience's benefit, use FIPS 199 and other Standards' terminology (for example, _security categorization_ terms should be "low-, moderate-, and high-impact"), rather than creating different terms. Clarify that the _security area categories_, "controlled, limited, and exclusion" relate to "risk-based PIV authentication mechanisms for different areas within a facility," per NIST SP 800-116. Eliminate confusion between the Playbook's two similar terms, _security categorization_ and _security area categories_.  SP 800-116, uses the term, "boundaries" (in one case), for "controlled, limited, and exclusion."
- Use bullets, steps, and checklists wherever possible (for example, convert the “_What Do I Need in My Site/Agency?_” information to a checklist and move to an upfront location in Playbook [location – TBD]).  
- Incorporate standard operating procedures (SOPs) and lessons learned from federal agencies
- Ensure compliance with federal policies and standards
- Update/modernize/clarify graphics using newer visuals and icons
- Combine tables that repeat similar information (for example, Tables 1-3)
- Remove references to deprecated documents (for example, FICAM Roadmap, Field Manual 3-19.30, etc.)
- Limit referenced documents lists in main body
- For referenced documents cited in main body, use only identifiers (e.g., NIST SP 800-116) and titles.  Full citations will be included in a "_Standards and Policies_" section.

#### Link to Additional Guidelines: ####

https://github.com/GSA/ficam-pacs/blob/staging/README.md

#### Links to the Content Pages for Contributors: ####

https://federalist-proxy.app.cloud.gov/preview/gsa/ficam-pacs/pacs-playbook/facilityrisk/

https://federalist-proxy.app.cloud.gov/preview/gsa/ficam-pacs/pacs-playbook/authmech/

**Comment by clstmbrly** [comment]
Corrected links:

**Link to Additional Guidelines:**

(https://github.com/GSA/ficam-pacs/blob/staging/README.md)

**Link to Navigation (Outline) Issue #2 :**

(https://github.com/GSA/ficam-pacs/issues/2)

**Links to the Content Page for Contributors:**

(https://federalist-proxy.app.cloud.gov/preview/gsa/ficam-pacs/pacs-playbook/facilityrisk/)

(https://federalist-proxy.app.cloud.gov/preview/gsa/ficam-pacs/pacs-playbook/authmech/)

**Comment by clstmbrly** [comment]
Per the new PACS navigation, these 2 sections have been combined:   “Determine Facility Risk” (previously "Selecting Determining Facility Risk") and “Select Authentication Mechanism” (previously "Selecting the Appropriate Authentication Mechanism").  The major section heading is now "Aligning Facility Security Level (FSL) and Authentication."

**Comment by markjdale** [comment]
This section is to be focused on authentication mechanisms at access points; (e.g., doors & gates) where an authentication mechanism for a "reader" is to be selected. There are 2 issues with the recommendations in this context:
1. Facility Security Level (as introduced by the ISC Risk Management Process) is a high-level designation that is determined for a facility based on 5factors: Mission Criticality, Symbolism, Facility Population, Facility Size, and Threat to Tenant Agencies.  The result of a Facility Security Level Determination is that a facility will be designated as an FSL I, FSL II, FSL II, FSL IV, and FSL V facility. This is not really helpful, since any FSL-level facility may have an area only requiring single factor authentication , and another area requiring 3-factor authentication, regardless of what FSL level they are. SP 800-116, 7.3
Selection of PIV Authentication Mechanisms,  even states that "There is no simple one-to-one mapping between the FSL and the authentication mechanism(s) that should be employed. .... For these reasons, it is recommended that authentication mechanisms be selected on the basis of protective areas established around assets or resources." Thus, it is proposed that we do away with any confusion with including/referencing FSL, and having readers educate themselves with the ISC FSL Determination process.
2. We should keep terms consistent and not introduce any new ones. Agreed. We should follow the recommendation and stick with "security area categories: controlled, limited, and exclusion", but not introduce FIPS-199 security impact categorizations (i.e., Low, Moderate, & High), as they relate to the IT systems themselves, not the access points to which the section is to address the selection of authentication mechanisms.

In short, remove FSL and FIPS 199 references and terminology.

**Comment by ryancdickson** [comment]
Comment adjudicated.
